cranelift/codegen/src/isa/x64/abi.rs

```diff
@@ -851,6 +853,7 @@ fn get_fltreg_for_retval(
     retval_idx: usize,
 ) -> Option<Reg> {
     match call_conv {
+        CallConv::Tail => unimplemented!(),
```

Comment:
Can you add this on the other architectures too?

Reply:
The other ISAs don't do exhaustive matches on the calling conventions, so there isn't anywhere to update.